packages/esm-appointments-app/src/routes.json

```diff
@@ -90,14 +90,6 @@
         "columnSpan": 1
       }
     },
-    {
-      "name": "patient-appointments-overview-widget",
-      "component": "appointmentsOverview",
-      "order": 8,
-      "meta": {
-        "columnSpan": 4
-      }
-    },
```

Comment:
See above, as far as I could tell this extension was not being used anywhere, so I repurposed it.